src/main/java/org/testng/internal/XmlMethodSelector.java

```diff
@@ -283,6 +290,13 @@ public void setIncludedGroups(Map<String, String> includedGroups) {
   }
 
   private static boolean isIncluded(String[] groups, Collection<String> includedGroups) {
+    return isIncluded(groups, includedGroups, false);
+  }
+
+  private static boolean isIncluded(String[] groups, Collection<String> includedGroups, boolean noGroupsSpecified) {
```

Comment:
What if we have only `private static boolean isIncluded(String[] groups, Collection<String> includedGroups, Collection<String> excludedGroups)` instead of 2 methods?